releases/lib/common/handler.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Update the dates in the headers.

Reply:
Since this is derivative of previous work in the repo, it should be based on the original dates.

See https://github.com/flutter/cocoon/pull/701#discussion_r395447338